src/js/features/arbitration/reducer.js

```diff
@@ -83,13 +83,26 @@ function reducer(state = DEFAULT_STATE, action) {
         }
       };
     case RESOLVE_DISPUTE_SUCCEEDED:
+    {
+      const arbitration = state.arbitration;
+      arbitration.arbitration.open = false;
+      arbitration.arbitration.result = action.result;
+
+      const currentEscrow = escrows.find(x => x.escrowId === action.escrowId);
+      if(currentEscrow) {
+        currentEscrow.arbitration.open = false;
+        currentEscrow.arbitration.result = action.result;
+      }
+
       return {
         ...state, ...{
           escrows,
+          arbitration,
```

Comment:
To not modify the state directly:
```suggestion
          arbitration: {
            ...state.arbitration,
            arbitration: {
              arbitration: {
                open: false,
                result:  action.result
              }
            }
          },
```
You can then get rid of the code above

Reply:
ahh! I always forget this. Sorry!